Source/Core/Core/ConfigManager.cpp

```diff
@@ -747,6 +747,16 @@ void SConfig::SetRunningGameMetadata(const std::string& game_id, u64 title_id, u
   m_title_id = title_id;
   m_revision = revision;
 
+  if (game_id.length() == 6)
+  {
+    m_debugger_game_id = game_id;
+  }
+  else if (title_id != 0)
+  {
+    m_debugger_game_id = StringFromFormat("%08X_%08X", static_cast<u32>(title_id >> 32),
+                                          static_cast<u32>(title_id));
+  }
```

Comment:
Should we add an `else { m_debugger_game_id.clear(); }` at the end so that old values won't linger around? Or is that going to mess with the way you're handling it for DOL/ELF files?